src/Discord.Net.WebSocket/Entities/Interaction/SocketInteraction.cs

```diff
@@ -96,97 +93,76 @@ internal virtual void Update(Model model)
         }
 
         /// <summary>
-        ///     Responds to an Interaction.
+        ///     Responds to an Interaction with type <see cref="InteractionResponseType.ChannelMessageWithSource"/>.
         /// <para>
         ///     If you have <see cref="DiscordSocketConfig.AlwaysAcknowledgeInteractions"/> set to <see langword="true"/>, You should use
-        ///     <see cref="FollowupAsync(Embed[],string, bool, bool, InteractionResponseType, AllowedMentions, RequestOptions, MessageComponent)"/> instead.
+        ///     <see cref="FollowupAsync(Discord.Embed[],string,bool,bool,Discord.AllowedMentions,Discord.RequestOptions,Discord.MessageComponent)"/> instead.
```

Comment:
Again, we should keep the type able to be specified

Reply:
Redundant and necessary

Reply:
Each response has its own method that accepts the necessary arguments for it